### ISSUE TITLE
Remove Workarounds for API-Levels < 15

### ISSUE THREAD

**Issue by eugenkiss** [eugenkiss/chanobol#187]
I decided to drop support for API-Levels < 15 because during implementation of in-app WebM playback I realized how much extra work I have to do to support API-Levels < 15 in parallel. There are also a couple of other bugs that simply vanish. Also, finally I can use a more flexible settings screen (settings fragment approach). If somebody sacrifices himself to maintain code for API < 15 I would rethink this issue but I won't do it myself.
